pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -78,12 +79,30 @@ type MonitorInfo struct {
 	Image string `json:"image"`
 }
 
+// PullPolicy describes a policy for if/when to pull a container image
+type PullPolicy string
+
+const (
+	// PullAlways means that kubelet always attempts to pull the latest image. Container will fail If the pull fails.
+	PullAlways PullPolicy = "Always"
```

Comment:
Can you please explain the use cases for this, Why are you defining them here?

Reply:
no need, removed in new commit